system-addon/content-src/components/Card/Card.jsx

```diff
@@ -31,7 +31,10 @@ class Card extends React.Component {
   }
   onLinkClick(event) {
     event.preventDefault();
-    this.props.dispatch(ac.SendToMain({type: at.OPEN_LINK, data: this.props.link}));
+    this.props.dispatch(ac.SendToMain({
+      type: at.OPEN_LINK,
+      data: Object.assign(this.props.link, {where: (event.ctrlKey || event.metaKey || event.button === 1) ? "tab" : "current"})
```

Comment:
Looks like you could call`browser.ownerGlobal.whereToOpenLink` with the event, if you can pass the whole event up